contributors.yml

```
@@ -1 +1,2 @@
 - mcansh
+- SCasarotto
```

Should be lowercase for some reason 🤷‍♂️

```suggestion
- scasarotto

```

---

ah shoot, i fixed this in the inverse 😂